### PR TITLE
Use expo-screen-orientation

### LINKED ISSUE #7: undefined is not an object: ScreenOrientation.default.Orientation
With Expo SDK 36:

undefined is not an object (evaluating '_ScreenOrientation.default.Orientation')
- node_modules/@expo/match-media/build/polyfill.native.js:7:8 in MediaQuery#constructor

file
image

### PR DESCRIPTION
**Pull request by ianmartorell**
Fixes #7.

Not sure if the changes to package.json are quite right, let me know if anything needs to be adjusted!

### CODE REVIEW

File: src/polyfill.native.ts [hunk 1]
```diff
@@ -9,14 +9,14 @@ class MediaQuery {
   private listeners: Listener[] = [];
 
   private orientation: ScreenOrientation.Orientation =
-    ScreenOrientation.Orientation.PORTRAIT;
+    ScreenOrientation.Orientation.PORTRAIT_UP;
```
Comment (NoemiRozpara) [inline src/polyfill.native.ts:12]: Shouldn't here be assigned just a string "portrait", as later we are asigning also a string? (L55-59)

Comment (ianmartorell) [reply src/polyfill.native.ts:12]: Ah yes definitely, it should be just `"portrait"`. It was working anyway when I tested it though, were you getting errors?

Comment (ianmartorell) [reply src/polyfill.native.ts:12]: Actually I just checked again and I think it's fine the way it is. We aren't assigning a string later in L55-59, we're passing the string to `mediaQuery.match` after comparing `this.orientation` to `ScreenOrientation.Orientation`.

**Comment by xiCO2k** [comment]
just left a tweet to @EvanBacon to check this pull request.

**Comment by EvanBacon** [comment]
Thanks for bringing this to my attention, GitHub notifications are not great. I fixed a few bugs and ran the tests, seems like everything is in order.

**Comment by xiCO2k** [comment]
That’s awesome thanks @EvanBacon

**Pull request merged**